src/huggingface_hub/hf_api.py

```diff
@@ -8261,6 +8277,173 @@ def grant_access(
         hf_raise_for_status(response)
         return response.json()
 
+    ###################
+    # Manage webhooks #
+    ###################
+
+    @validate_hf_hub_args
+    def get_webhook(self, webhook_id: str, token: Optional[str] = None) -> Dict:
```

Comment:
```suggestion
    def get_webhook(self, webhook_id: str, token: Optional[str] = None) -> Webhook:
```

=> would be good to type the output value with a dataclass (called `Webhook`?). This way users will know which fields to expect and IDEs will support autocompletion.

This meant the `return response.json()` must be adapted as well.

Reply:
Same comment for `list_webhooks`, `create_webhook`, `update_webhook`